authentik/sources/saml/models.py

```diff
@@ -200,6 +200,9 @@ class SAMLSource(Source):
         default=RSA_SHA256,
     )
 
+    signed_assertion = models.BooleanField(default=False)
```

Comment:
```suggestion
    signed_assertion = models.BooleanField(default=True)
```

for backwards compat, this is the current behaviour (only checked when verification keypair is selected anyways)